binding.gyp

```diff
@@ -92,10 +92,12 @@
       "include_dirs" : [
         "/usr/local/Cellar/ffmpeg@5/5.1.3/include/",
         "/opt/homebrew/Cellar/ffmpeg@5/5.1.3/include/",
+        "/opt/homebrew/Cellar/ffmpeg@5/5.1.4_4/include/",
```

Comment:
homebrew does not allow to switch to @5 5.1.3 which is anoying. I tried to find a rb of this version, checksum validation fails for me. This is the easiest way to run it on my machine and not to break other